### ISSUE TITLE
pgp key expired

### ISSUE THREAD

**Issue by peteobrien** [mumble-voip/mumble#3317]
hi guys, just to let you know seems your gpg key has expired for latest DMG mac build at least

**Comment by mkrautz** [comment]
Just to clarify, it has expired, but it was valid at the time of signing.
We've yet to release builds in 2018, so there is no real "problem" yet.
But I'll make sure to create a new key soon.

**Comment by peterobrien** [comment]
Thanks, and thanks for the software :)
New to the PGP process verification TBH. GPG doesn't seem to allow any verification of an expired key. Whilst I'm sure it's still valid, I wasn't able to verify that it was valid at the time of signing...

**Comment by mkrautz** [comment]
No worries. It's worth adding here that .dmg and .msi releases are already signed. The PGP signature is just icing on the cake.

The .dmgs are signed with Apple's Developer ID code signing, and our .msi releases are Authenticode signed.

tl;dr: We use Windows and macOS's native code signing facilities. For those platforms, PGP is just an extra layer, for people who like that.

**Comment by peterobrien** [comment]
Cool, thanks, I wasn't worried, but I also wasn't aware it was developer
signed. This adds credibility for sure. Thanks for the brilliant software

**Comment by mkrautz** [comment]
Fixed via https://github.com/mumble-voip/mumble-gpg-signatures/commit/91ce07ef784febc40811450ed1c58454d9b26032.
